sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -674,7 +677,7 @@ namespace Azure { namespace Core { namespace Http {
      *
      * @throw if \p header has an invalid header name or if the delimiter is missing.
      */
-    void AddHeader(std::string const& header);
+    void SetHeader(std::string const& header);
```

Comment:
Same here, update docs, talking about what happens if the header key already exists.